specification/peering/resource-manager/Microsoft.Peering/stable/2021-01-01/peering.json

```diff
@@ -0,0 +1,3540 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-01-01",
+    "title": "PeeringManagementClient",
+    "description": "APIs to manage Peering resources through the Azure Resource Manager."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/cdnPeeringPrefixes": {
+      "get": {
+        "tags": [
+          "CdnPeeringPrefixes"
+        ],
+        "description": "Lists all of the advertised prefixes for the specified peering location",
+        "operationId": "CdnPeeringPrefixes_List",
+        "parameters": [
+          {
+            "name": "peeringLocation",
+            "in": "query",
+            "description": "The peering location.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CdnPeeringPrefixListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all the cdn peering prefixes advertised at a particular peering location": {
+            "$ref": "./examples/ListCdnPeeringPrefixes.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/CheckServiceProviderAvailability": {
+      "post": {
+        "tags": [
+          "CheckServiceProviderAvailability"
+        ],
+        "description": "Checks if the peering service provider is present within 1000 miles of customer's location",
+        "operationId": "CheckServiceProviderAvailability",
```

Comment:
This is typically of the form [OperationGroup]_[Method], is the purpose of this api to see if this provider is available under the given {subscriptionId}?  If so is there a reason we can't use the Providers API given by ARM to do this?